### DISCUSSION TITLE
Audit logs automatically deleted at some point?

### DISCUSSION THREAD

**Discussion by ketalabama** [sheltermanager/asm3#1532] [Q&A / Help] [answered]
What is the policy around keeping audit trail intact for users/animal Details? I have not been able to figure out if they get deleted after a set X months or after X period of inactivity? But it is distressing to try to look back over audit history for an animal and it have been deleted. If there is a setting I am missing with respect to how long to keep audit please let me know.

**Comment by bobintetley** [comment, marked as answer]
Hi, audit items are deleted after 182 days (6 months). It cannot be configured. This is done on purpose, because the audit table is written to for every action done in the system and read for every record opened, it needs to not get too large or it would negatively impact system performance. Its purpose is for identifying who did what in case of errors or malicious activity within that 6 month window.

Who created or last changed a record is kept in fields on every record, so you can always see that info.

**Reply by ketalabama** [reply]
Thank you
